infra/experimental/chronos/cloudbuild.yaml

```diff
@@ -0,0 +1,47 @@
+# CloudBuild for generating Chronos-cached images.
+steps:
+- name: ubuntu
+  args:
+  - infra/experimental/chronos/prepare-recompile
+  - ${_PROJECT}
+  - ${_FUZZ_TARGET}
+  - ${_FUZZING_LANGUAGE}
+- name: 'gcr.io/cloud-builders/docker'
+  args:
+  - build
+  - -t
+  - gcr.io/oss-fuzz/${_PROJECT}
```

Comment:
Did we decide to use a different container registry (e.g., `oss-fuzz-gen`)?

Reply:
yep, you're right. pushing to `us-central1-docker.pkg.dev/oss-fuzz/oss-fuzz-gen/` instead.